cmd/builder/internal/builder/templates/main.go.tmpl

```diff
@@ -11,6 +11,9 @@ import (
 	{{- range .Providers}}
 	{{.Name}} "{{.Import}}"
 	{{- end}}
+	{{- range .Converters}}
+	{{.Name}} "{{.Import}}"
+	{{- end}}	
```

Comment:
nit: Add them before providers (since most likely will be in `confmap/converters`).

Reply:
thanks, done